GetImpersonateService/function.json

```diff
@@ -0,0 +1,20 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "route": "adm/impersonate-service/{serviceId}",
+      "methods": [
+        "get"
+      ]
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "res"
+    }
+  ],
+  "scri++ptFile": "../dist/GetImperonateService/index.js"
```

Comment:
The word script is spelled wrong

Reply:
Fixed in [cb5b0cc](https://github.com/pagopa/io-functions-admin/pull/174/commits/cb5b0cc8dc89fada47d29b8593d9fb806cc8a81f)